Add constraints around hawkular and heapster

wshearn opened the pull request.

rhdedgar committed:
adding memcached sso container

rhdedgar committed:
fixing roles symlink

rhdedgar committed:
tested on ops

wshearn committed:
Update logging installer

drewandersonnz committed:
use image uuid instead of image name

Commit:
router monitoring

add script to check health on each individual router pod
also report on whether the number of running pods (for each router DC) matches the definition in the DC

wshearn committed:
Add constraints around hawkular and heapster

twiest reviewed (approved):
These look fine to me, but I honestly know very little about the metrics deploy roles.

@kwoodson would you mind taking a look too?

kwoodson commented on ansible/roles/openshift_metrics/tasks/main.yml:
```diff
@@ -103,36 +103,92 @@
+    seconds: 30
+  run_once: true
+
+- name: Add the resource constraints to the hawkular RC
```
Did you mean heapster?

kwoodson commented on ansible/roles/openshift_metrics/tasks/main.yml:
```diff
@@ -103,36 +103,92 @@
+      spec.template.spec.containers[0].resources.limits.memory: "{{ osamet_heapster_limits_memory }}"
+      spec.template.spec.containers[0].resources.requests.memory: "{{ osamet_heapster_requests_memory }}"
+
+- name: scale up hawkular to 1
```
Did you mean heapster?

kwoodson commented on ansible/roles/openshift_metrics/tasks/main.yml:
```diff
@@ -103,36 +103,92 @@
+
+- name: scale up hawkular to 1
+  oc_scale:
+    name: "heapster"
```
Don't need quotes on name.

kwoodson commented on ansible/roles/openshift_metrics/tasks/main.yml:
```diff
@@ -103,36 +103,92 @@
+- name: Add the resource constraints to the hawkular RC
+  oc_edit:
+    kind: rc
+    name: "heapster"
```
don't need quotes on name.

kwoodson commented on ansible/roles/openshift_metrics/tasks/main.yml:
```diff
@@ -103,36 +103,92 @@
+    namespace: openshift-infra
+    replicas: 0
+
+- name: Wait for hawkular to scale down
```
did you mean heapster?

kwoodson commented on ansible/roles/openshift_metrics/tasks/main.yml:
```diff
@@ -103,36 +103,92 @@
+
+- name: scale up hawkular to 1
+  oc_scale:
+    name: "hawkular-metrics"
```
no need for quotes

kwoodson commented on ansible/roles/openshift_metrics/tasks/main.yml:
```diff
@@ -103,36 +103,92 @@
+
+- name: scale down heapster to 0
+  oc_scale:
+    name: "heapster"
```
no need for quotes.

kwoodson reviewed (commented):
Few suggestions.

Commit:
Fixing a bug with the yedit separator and _replace_content.

mwoodson committed:
fixed some output of package_update_needed

mwoodson committed:
updated new version of 3.3 vendored

Commit:
Adding entitlements back for builds.

mwoodson committed:
changed the package cleanup option

Commit:
Fix for zabbix maintenance.

Commit:
enable extras repo in ops-base

just because the host has a repo enabled, doesn't mean the container has it enabled. so explicitly enable it during build.

Commit:
Automatic commit of package [python-openshift-tools] release [0.0.95-1].

Commit:
Automatic commit of package [openshift-tools-scripts] release [0.0.166-1].

Commit:
add router monitoring script to scripts-monitoring-openshift RPM

Commit:
Automatic commit of package [openshift-tools-scripts] release [0.0.167-1].

Commit:
Automatic commit of package [openshift-tools-ansible] release [0.0.30-1].

drewandersonnz committed:
retry curl of not successful

drewandersonnz committed:
logger.debug curlCount

drewandersonnz committed:
curl function tidy-up

BlueShells committed:
Automatic commit of package [openshift-tools-scripts] release [0.0.168-1].

drewandersonnz committed:
Automatic commit of package [openshift-tools-scripts] release [0.0.169-1].

wshearn committed:
Minor formatting changes
